src/comm/USBBoardInfo.json

```diff
@@ -13,6 +13,7 @@
         { "vendorID": 9900, "productID": 33,        "boardClass": "Pixhawk",    "name": "AUAV X2.1 FMU V2" },
         { "vendorID": 9900, "productID": 48,        "boardClass": "Pixhawk",    "name": "MindPX FMU V2" },
         { "vendorID": 9900, "productID": 50,        "boardClass": "Pixhawk",    "name": "PX4 FMU V5" },
+        { "vendorID": 9900, "productID": 51,        "boardClass": "Pixhawk",    "name": "PX4 FMU V5X" },
```

Comment:
@mrpollo @LorenzMeier The bootloader that has been out for years use 12677

Reply:
Thanks, I sent the following https://github.com/mavlink/qgroundcontrol/pull/9816